geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/KafkaDataStore.scala

```diff
@@ -86,6 +86,8 @@ object KafkaDataStoreFactoryParams {
   val IS_PRODUCER_PARAM  = new Param("isProducer", classOf[java.lang.Boolean], "Is Producer", false, false)
   val EXPIRATION_PERIOD  = new Param("expirationPeriod", classOf[java.lang.Long], "Features will be auto-dropped (expired) after this delay in milliseconds. Leave blank or use -1 to not drop features.", false)
   val CLEANUP_LIVE_CACHE = new Param("cleanUpCache", classOf[java.lang.Boolean], "Run a thread to clean up the live feature cache every second if set to true. False by default.", false)
+  val MONITOR            = new Param("monitor", classOf[java.lang.Boolean], "Enable monitoring stats for feature store.", false)
```

Comment:
can we think of a better name for this? in accumulo we use 'collectQueryStats' - we should probably try for consistency. (i wouldn't mind renaming the accumulo one if you come up with something and we can just keep the old name for back compatibility)